README.md

```diff
@@ -18,6 +18,16 @@ Note: docker registry must be [v2](https://docs.docker.com/registry/spec/api/).
 
 * `password`: *Optional.* The password to use when authenticating.
 
+* `login`: *Optional.* An array of additional username, password and registry
```

Comment:
Can we call this `additional_login` or something to prevent confusion with `username` and `password`?